CredentialProvider.Microsoft/Util/SessionTokenCache.cs

```diff
@@ -23,6 +23,7 @@ public SessionTokenCache(string cacheFilePath, ILogger logger, CancellationToken
         {
             this.cacheFilePath = cacheFilePath;
             this.logger = logger;
+            this.cancellationToken = cancellationToken;
```

Comment:
Looked like this was missing. There's still oddness though in that the request handler interface in nuget requires a token both as a member and as a parameter. I think we're getting default tokens in some places we shouldn't - worth looking at sometime.